package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "check-dependency-version-consistency",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
I have an automated release process with `yarn release` that will be done separately from PRs.